sixtyfps_compiler/passes/default_geometry.rs

```diff
@@ -199,19 +213,11 @@ fn make_default_100(elem: &ElementRc, parent_element: &ElementRc, property: &str
     });
 }
 
-fn make_default_implicit(elem: &ElementRc, property: &str) {
+fn make_default_implicit(elem: &ElementRc, property: &str, orientation: Orientation) {
```

Comment:
Just a minor thought: I suppose this function could be changed to just take the orientation (and element of course) and derive the property name from the Orientation, since it's always either width or height.

Reply:
there is only two call to this internal function. So i think it's fine because we would have to do a match there instead of just specifying it.